velox/functions/prestosql/tests/RepeatTest.cpp

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::test;
+using namespace facebook::velox::functions::test;
+
+namespace {
+
+class RepeatTest : public FunctionBaseTest {
+ protected:
+  void testRepeat(
+      const VectorPtr& expected,
+      const std::vector<VectorPtr>& input) {
+    auto result = evaluate<ArrayVector>("repeat(C0, C1)", makeRowVector(input));
+    assertEqualVectors(expected, result);
+  }
+
+  void testTryRepeat(
+      const VectorPtr& expected,
+      const std::vector<VectorPtr>& input) {
+    auto result =
+        evaluate<ArrayVector>("try(repeat(C0, C1))", makeRowVector(input));
+    assertEqualVectors(expected, result);
+  }
+
+  void testError(
+      const std::vector<VectorPtr>& input,
+      const std::string& expectedError) {
+    VELOX_ASSERT_THROW(
+        evaluate<ArrayVector>("repeat(C0, C1)", makeRowVector(input)),
+        expectedError);
+  }
+};
+} // namespace
+
+TEST_F(RepeatTest, repeatInteger) {
```

Comment:
The implementation of the function doesn't depend on the type of the array elements, hence, no need to add tests for different types. At the time, the most common use case is when 'count' is a constant, but there are not test cases for that.